### PR TITLE
Update Daft to 0.2.13

### PR DESCRIPTION
**Pull request by rkenmi**

### CODE REVIEW

**Comment by yankevn** [comment]
Did you run any test compaction jobs with this change?

**Comment by rkenmi** [comment]
> Did you run any test compaction jobs with this change?

✅

**Review by yankevn — APPROVED** [review]
LGTM

File: setup.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def find_version(*paths):
         "typing-extensions == 4.4.0",
         "pymemcache == 4.0.0",
         "redis == 4.6.0",
-        "getdaft == 0.2.4",
+        "getdaft == 0.2.13",
```
Comment (raghumdani) [inline setup.py:50]: What about changing requirements?

Comment (rkenmi) [reply setup.py:50]: Separate PR: https://github.com/ray-project/deltacat/pull/262

**Pull request merged**